packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Helper method to determine which comparison results are relevant if the screenshot assertion
+ * fails.
+ *
+ * @param {Object} screenshot - The comparison result
+ * @param {boolean} screenshot.isSameDimensions - If the latest screenshot was the same size as the reference screenshot.
+ * @param {boolean} screenshot.isWithinMismatchTolerance - If the latest screenshot was within the mismatch tolerance.
+ * @param {boolean} screenshot.isExactSameImage - If the latest screenshot matched the reference screenshot exactly, i.e. 0% mismatch.
+ * @param {Number} screenshot.misMatchPercentage - The mismatch percentage when comparing the latest screenshot to the reference screenshot.
+ * @returns {String} - A message describing the status of the screenshot.
+ */
+function getComparisonResult(screenshot) {
+  const { misMatchPercentage, isSameDimensions } = screenshot;
+  const relevantInformation = {};
+
+  if (!isSameDimensions) {
+    relevantInformation.isSameDimensions = isSameDimensions;
+  }
+
+  relevantInformation.misMatchPercentage = misMatchPercentage;
+
+  return `${JSON.stringify(relevantInformation, null, 2)}`;
+}
+
+/**
+ * An assertion method to be paired with Visual Regression Service to assert each screenshot is within
+ * the mismatch tolerance and are the same size.
+ *
+ * @param {Object} screenshot - The comparison result
+ * @param {boolean} screenshot.isSameDimensions - If the latest screenshot was the same size as the reference screenshot.
+ * @param {boolean} screenshot.isWithinMismatchTolerance - If the latest screenshot was within the mismatch tolerance.
+ * @param {boolean} screenshot.isExactSameImage - If the latest screenshot matched the reference screenshot exactly, i.e. 0% mismatch.
+ * @param {Number} screenshot.misMatchPercentage - The mismatch percentage when comparing the latest screenshot to the reference screenshot.
+ * @returns {Object} - An object that indicates if the assertion passed or failed with a message.
+ */
+function toMatchReference(receivedScreenshot) {
```

Comment:
nit: Param documentation doesn't align. `screenshot`/`receivedScreenshot `. 

I'm on the fence about documenting values that aren't used, but it doesn't hurt, it's just a little confusing at first. Several values aren't used in some of these functions.

Reply:
>  Several values aren't used in some of these functions.

Which values? If there are that aren't being used in our screenshot assertion, we should remove them from the comparison results that are generated by the screenshot strategy.

Reply:
Anywhere the `screenshot` object is referenced every property of that object is documented, which sounds good in practice, but is slightly confusing when looking at the purpose of the function. 

`getComparisonResult ` for example documents 4 properties (5 counting the root object), but only uses `misMatchPercentage` and `isSameDimensions`.

I don't think it's inherently bad to keep the documentation

Reply:
Updated to only listed out the properties that are used in the method. `isExactSameImage` is not being used so it is removed here and in BaseCompare.js.

https://github.com/cerner/terra-toolkit/pull/545/commits/9933873194e2d7e6d395853f313cdd4cf89d1212